magefiles/magefile.go

```diff
@@ -51,7 +51,7 @@ var (
 	// can be periodic, presubmit or postsubmit
 	jobType                    = utils.GetEnv("JOB_TYPE", "")
 	reposToDeleteDefaultRegexp = "jvm-build|e2e-dotnet|build-suite|e2e|pet-clinic-e2e|test-app|e2e-quayio|petclinic|test-app|integ-app|^dockerfile-|new-|^python|my-app|^test-|^multi-component"
-	repositoriesWithWebhooks   = []string{"devfile-sample-hello-world", "hacbs-test-project"}
+	repositoriesWithWebhooks   = []string{"devfile-sample-hello-world", "hacbs-test-project", "secret-lookup-sample-repo-two"}
```

Comment:
Why did we not add "secret-lookup-sample-repo-one" here?

Reply:
[according to the test code](https://github.com/redhat-appstudio/e2e-tests/blob/27b1fbf8af6ea4968f525c03a9db3087d5fb9308/tests/build/build.go#L994) that repo is used for running a negative test, i.e. it is not expected that any PLR will be created there
I'm happy to add it there, but I think it's not needed... not sure what @tisutisu thinks

Reply:
@psturc you are right. Its a error scenario for `secret-lookup-sample-repo-one` , so no need to add here.